src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-FSharp/Company.WebApplication1.fsproj

```diff
@@ -15,10 +15,10 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.0.0" Condition="'$(IncludeApplicationInsights)' == 'True'" />
     <PackageReference Include="Microsoft.AspNetCore" Version="1.0.7" Condition="'$(Framework)' != 'netcoreapp1.1'" />
-    <PackageReference Include="Microsoft.AspNetCore" Version="1.1.4" Condition="'$(Framework)' == 'netcoreapp1.1'" />
+    <PackageReference Include="Microsoft.AspNetCore" Version="1.1.5" Condition="'$(Framework)' == 'netcoreapp1.1'" />
```

Comment:
This file looks odd compared to the other files. Every other project has two separate dependency sections for 1.0 and 1.1. However, the only PackageReference that's different between 1.0 and 1.1 for this project is `Microsoft.AspnetCore`. Does this mean that the 1.0 fsproj also references Microsoft.AspNetCore.Mvc 1.1.6 and Microsoft.Extensions.Logging.Debug 1.1.2?